app/src/main/java/com/sdp/movemeet/Backend/Serialization/ActivitySerializer.java

```diff
@@ -44,15 +44,16 @@ public Activity deserialize(Map<String, Object> data) {
                 (double )data.get(LATITUDE_KEY),
 
                 (String) data.get(DESC_KEY),
+                (String) data.get(DOCUMENT_PATH_KEY),
                 (Date) data.get(DATE_KEY),
                 (double) data.get(DURATION_KEY),
                 (Sport) data.get(SPORT_KEY),
                 (String) data.get(ADDRESS_KEY),
                 (Date) data.get(CREATION_KEY)
         );
 
-        String path = (String) data.get(DOCUMENT_PATH_KEY);
-        if (path != null) act.setDocumentPath(path);
+//        String path = (String) data.get(DOCUMENT_PATH_KEY);
```

Comment:
Why is this commented ?

Reply:
Good catch! This is because we don't need this line anymore. This week I modified the constructor of the Activity class and directly integrated the "documentPath" in it. We hence don't need to set the Firebase Storage "documentPath" afterwards (i.e. after creation) to the Activity object. By the way, I really needed to have an easy access to the "documentPath" to retrieve the information related to the sport activity (among others the ID of the participants).